tcf_website/tests/test_utils.py

```diff
@@ -96,7 +161,7 @@ def setup(obj):
         amount_group=20,
         amount_reading=20,
         amount_writing=20,
-        amount_homework=20
+        amount_homework=20,
```

Comment:
I don't usually tell people to use or not use trailing commas (although I always use them when a list of items spans multiple lines), but this seems inconsistent given that you didn't use trailing commas elsewhere (e.g. `obj.instructor_grade`). I think this diff is unnecessary, unless you want to enforce a consistent coding style for _all_ such cases.
```suggestion
        amount_homework=20
```
I personally believe minimizing diffs helps future developers maintain the same code base because it reduces the number of diff they have to check. If there are too many unnecessary diffs in a commit/PR (although that's not the case in this PR), one is less likely to look closely into them.
If you do commit this suggestion, make sure to change `obj.review2` as well.